extensions/common/vault/vault-gcp/src/test/java/org/eclipse/edc/vault/gcp/GcpSecretManagerVaultExtensionTest.java

```diff
@@ -57,8 +58,10 @@ void noSettings_shouldThrowException() {
         when(invalidContext.getMonitor()).thenReturn(monitor);
         when(invalidContext.getConfig()).thenReturn(ConfigFactory.empty());
 
+        extension.gcpManager = new GcpManager(invalidContext);
+
         EdcException exception = assertThrows(EdcException.class, () -> extension.initialize(invalidContext));
-        assertEquals("No setting found for key edc.vault.gcp.region", exception.getMessage());
+        assertEquals("No setting found for key " + GcpSecretManagerVaultExtension.VAULT_REGION, exception.getMessage());
```

Comment:
not directly part of the PR, but assertions should be done with assertj (assertThat...), for consistency

Reply:
Will update it

Reply:
Done